docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -216,7 +216,7 @@ A JSON array containing the updated rules.
 [[bulk-actions-rules-api-action]]
 ==== Bulk action
```

Comment:
This is an existing page question, but any reason `Create`/`Update` have the `this API supports Token-based authentication` warning 
<p align="center">
  <img width="350" src="https://user-images.githubusercontent.com/2946766/156076269-6193bcb9-05a9-455d-a835-44586c2c91d0.png" /> <img width="350" src="https://user-images.githubusercontent.com/2946766/156076382-dda37bfd-9fbe-472b-ab06-1e8ee94b6fd1.png" />
</p>



but `Delete`/`Action` do not?
<p align="center">
  <img width="350" src="https://user-images.githubusercontent.com/2946766/156076452-b1948406-1d66-40e8-8aa4-719709c60f8e.png" /> <img width="350" src="https://user-images.githubusercontent.com/2946766/156076536-92a9c5a2-135b-4109-99ed-db20a1413d82.png" />
</p>

It's the same for the whole API, so can this just be a page-level warning?

Reply:
@spong, not sure about this. Is there a way to determine which type of authentication is supported by particular API(maybe looking into route configuration)? 

Take for example this one, read rule https://www.elastic.co/guide/en/security/current/rules-api-get.html
and create rule https://www.elastic.co/guide/en/security/current/rules-api-create.html#opt-fields-all

Create API has warning, but by looking into route configuration I don't see the difference. We might want to align these warnings to be consistent across different APIs

Reply:
> @spong, not sure about this. Is there a way to determine which type of authentication is supported by particular API(maybe looking into route configuration)?

I believe so long as security is enabled (which it is by default now) all routes will go through this auth ([source](https://github.com/elastic/kibana/blob/6627bd8b3ab49415e397495d8e076f9861fcc894/x-pack/plugins/security/server/authentication/providers/token.ts#L101-L106)).

So yeah, I think it's just a consistency thing -- generally, auth shouldn't vary between these Kibana CRUD routes, so I think we just need to align on these warnings like you said.